tck/index.adoc

```diff
@@ -14,215 +14,3483 @@ There is also an `uncategorized` and `precategorized` directory containing uncat
 === Call
 
 * Call1 - Basic procedure calling
+
```

Comment:
what is the reason for the empty lines?

Reply:
That is indeed weird. Good catch. Needs a fix in the generator. I will have look.

Reply:
fixed